### ISSUE TITLE
November 2023 newsletter

### ISSUE THREAD

**Issue by finekatie** [uswds/uswds-team#295]
Create and publish [November 2023 Newsletter](https://docs.google.com/document/d/1gaBdvUkR4YuwIRx8__ubyjVDQnXTWtaI_5chCB7tn78/edit). 
Deadline for publish: 11/9 or 11/13 (holiday on 11/10)

Helpful links:

- [Newsletter folder in Google Drive](https://drive.google.com/drive/folders/1xq1tcxcV5hZxNA-6_PH8pycQkkbrCXJ6)
- [GovDelivery](https://admin.govdelivery.com/session/new)

- [x] - [ ] Draft copy
  - [x] Make sure accessibility feature is ready. (Nov. accessibility feature should highlight WCAG 2.2 and [this GitHub discussion about it](https://github.com/uswds/uswds/discussions/5569).)
  - [x] Make sure release notes are available to link to
  - [x] Write summary of release for newsletter ([Amy L. got us started in Slack](https://gsa-tts.slack.com/archives/C050HRGN7/p1698855712471279) - see comment on thread from Charlie, too)
  - [x] Make sure [monthly call event description](https://docs.google.com/document/d/1S0dAQ9BoR0dgS55gF_VX00da8KO-l6Ob66owl3avRt0/edit) is ready (ping Dan to check in)
  - [x] New USWDS websites (See [this thread from Slack](https://gsa-tts.slack.com/archives/C050HRGN7/p1697135330137539) for a couple of new sites, as well as [this one](https://gsa-tts.slack.com/archives/C050HRGN7/p1696433936050749) and [this one](https://gsa-tts.slack.com/archives/C050HRGN7/p1696350268447979))
  - [x] other content as needed (ask team if previously mentioned content items are sparse)  
  - [x] important issues or announcements (will come to us from Anne/Dan, as determined)
- [x] Get feedback from content team
- [x] Get feedback from larger team - it can be especially helpful to to have the dev team review your edits to their release notes copy if you are unsure that we're getting the technical details right
- [x] Approval: Anne and Dan
- [x] Add content to GovDelivery
- [x] Proofread
- [x] Publish

**Comment by finekatie** [comment]
Katie and Sarah wrapped initial draft and shared with Dan and Anne.

**Comment by finekatie** [comment]
Still waiting for Anne and Dan approval. No reply from either yet in Slack about when they'll review.

**Comment by finekatie** [comment]
We have Dan's approval. Just waiting for Digital.gov to push up the link for the event page and then we can publish.

**Comment by finekatie** [comment]
Last night we got the link to the event page, updated and did last looks, and then scheduled for this morning.

**Comment by finekatie** [comment]
This is published and complete!
